feature/communicate/src/main/java/com/cybozu/sample/kintone/spaces/feature/communicate/thread/ThreadScreen.kt

```diff
@@ -56,11 +63,72 @@ fun ThreadScreen(
         ),
 ) {
     val uiState by viewModel.uiState.collectAsState()
+    val snackbarHostState = remember { SnackbarHostState() }
+    val isRefreshing = uiState is ThreadUiState.Loading
+    val swipeRefreshState = rememberSwipeRefreshState(isRefreshing = isRefreshing)
 
-    ThreadContent(
-        threadName = threadName,
-        uiState = uiState
-    )
+    LaunchedEffect(uiState) {
+        if (uiState is ThreadUiState.Error) {
+            snackbarHostState.showSnackbar((uiState as ThreadUiState.Error).errorMessage)
+        }
+    }
+    Scaffold(
+        topBar = {
+            TopAppBar(
+                title = {
+                    Text(threadName)
+                },
+                navigationIcon = {
+                    SystemBackNavButton()
+                }
+            )
+        },
+        snackbarHost = { SnackbarHost(snackbarHostState) }
+    ) { innerPadding ->
+        Box(
+            modifier =
+                Modifier
+                    .fillMaxSize()
+                    .padding(innerPadding),
+            contentAlignment = Alignment.Center
+        ) {
+            when (uiState) {
+                is ThreadUiState.Initial -> {
+                    Text("Initial State")
+                }
+                is ThreadUiState.Loading -> {
+                    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+                        CircularProgressIndicator()
+                        Spacer(modifier = Modifier.height(16.dp))
+                        Text("Loarding...")
```

Comment:
[Good]
ロード中、インジケータだけじゃなくてテキストでロード状態を伝えるテキストを出すのはユーザに優しくていいですね！

（たまたま誤字見つけました）
```suggestion
                        Text("Loading...")
```